pkg/stub/handler.go

```diff
@@ -175,7 +166,19 @@ func (h *Handler) ensureDNSNamespace() error {
 // ensureCoreDNSForClusterDNS ensures all necessary CoreDNS resources exist for
 // a given clusterdns.
 func (h *Handler) ensureCoreDNSForClusterDNS(dns *dnsv1alpha1.ClusterDNS) error {
-	ds, err := h.ManifestFactory.DNSDaemonSet(dns)
+	clusterIP := ""
+	// TODO: fetch this from higher level openshift resource when it is exposed
+	clusterDomain := "cluster.local"
+
+	if dns.Name == "default" {
```

Comment:
I'm ambivalent about special-casing on the clusterdns's name.  For now, clusterdns is a singleton, right? Does it make sense to special-case the name now? Will it make sense to special-case the name in the future, if we expand the API to support multiple clusterdnses?